Presentation.Web/Web.config

```diff
@@ -262,4 +271,41 @@
       </listeners>
     </trace>
   </system.diagnostics>-->
+  <Federation xmlns="urn:dk.nita.saml20.configuration" sessionType="dk.nita.saml20.Session.InProcSessionStoreProvider, dk.nita.saml20">
+    <SessionTimeout>30</SessionTimeout>
+    <PreventOpenRedirectAttack>true</PreventOpenRedirectAttack>
+    <SigningCertificate findValue="564a1e99402a66282e11fb7ef6b034ad6ab6b879" storeLocation="LocalMachine" storeName="My" x509FindType="FindByThumbprint"/>
+    <MetaDataShaHashingAlgorithm>SHA256</MetaDataShaHashingAlgorithm>
+    <AllowedAudienceUris>
+      <Audience>https://kitos.dk</Audience>
+    </AllowedAudienceUris>
+    <Actions>
+      <clear/>
+      <add name="SetSamlPrincipal" type="dk.nita.saml20.Actions.SamlPrincipalAction, dk.nita.saml20 "/>
+      <add name="Redirect" type="dk.nita.saml20.Actions.RedirectAction, dk.nita.saml20"/>
+    </Actions>
+  </Federation>
+  <SAML20Federation xmlns="urn:dk.nita.saml20.configuration" xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata">
+    <MinimumAssuranceLevel>2</MinimumAssuranceLevel>
+    <ShowError>true</ShowError>
+    <ServiceProvider id="https://kitos.dk" server="https://localhost:44300">
+      <ServiceEndpoint localpath="Login.ashx" type="signon" redirectUrl="SSO" index="0"/>
+      <ServiceEndpoint localpath="Logout.ashx" type="logout" redirectUrl="SSO" index="1"/>
+      <ServiceEndpoint localpath="Logout.ashx" type="soaplogout" errorBehaviour="throwexception"/>
+      <ServiceEndpoint localpath="Metadata.ashx" type="metadata"/>
+      <md:ContactPerson contactType="administrative">
+        <md:Company>STRONGMINDS</md:Company>
+        <md:EmailAddress>admin@strongminds.dk</md:EmailAddress>
+      </md:ContactPerson>
+    </ServiceProvider>
+    <RequestedAttributes>
+      <att name="urn:oid:0.9.2342.19200300.100.1.1" isRequired="true"/>
+      <!--sAMAcountName, AD username-->
+    </RequestedAttributes>
+    <NameIdFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:X509SubjectName</NameIdFormat>
+    <IDPEndPoints metadata="idp-metadata">
+      <add id="https://saml.adgangsstyring.eksterntest-stoettesystemerne.dk" ShaHashingAlgorithm="SHA256" default="true"/>
+    </IDPEndPoints>
+    <CommonDomain enabled="false" localReaderEndpoint="https://pfs04/demo/cdcreader.ashx"/>
```

Comment:
hvad gør den linje godt for? skal den ikke bare ud? Den har noget med demoprojektet at gøre

Reply:
Den benytter OIOSAML.Net til at videregive Authnrequest til?

Reply:
men den er jo disabled... kan den fjernes? Der står "demo" i den så måske det er noget der knytter sig til det demo projekt denne fil blev baseret på